docs/administration-zk-bk.md

```diff
@@ -268,7 +274,12 @@ In Pulsar, you can set *persistence policies* at the namespace level, which dete
 
 You can set persistence policies for BookKeeper at the [namespace](reference-terminology.md#namespace) level.
 
-#### Pulsar-admin
+````mdx-code-block
```

Comment:
Double check: have you previewed line 277 - 388 to make everything go as expected?

Reply:
![image](https://github.com/apache/pulsar-site/assets/61108539/a190cef7-40e3-433b-80e2-bdbb33b840dc)
![image](https://github.com/apache/pulsar-site/assets/61108539/3c479e63-c010-40a6-8f7e-75e73e9feffd)
![image](https://github.com/apache/pulsar-site/assets/61108539/0f2be743-0ffa-4c89-afaa-1349dab6ddb2)